clients/mobile/src/donationWorkflow/donationPosts/PostCard.tsx

```diff
@@ -7,18 +7,28 @@ import { Button } from '../../components/button/Button'
 import { DonationData } from './useDonationPosts'
 import { useUserProfile } from '../../userWorkflow/context/UserProfileContext'
 
+const URGENT = 'urgent'
```

Comment:
This is a weird use if we change the string in the future. we have to change it in multiple places